examples/tfengine/full.yaml

```diff
@@ -119,15 +121,16 @@ templates:
 
 # Prod central apps project for team 1.
 - recipe_path: "{{$BASE}}/folder/project.yaml"
-  output_path: "./live/folder.prod/folder.team1"
+  output_path: "./live/prod/team1"
   data:
-    PROJECT_ID: "example-prod-apps"
-    APIS:
-    - "container.googleapis.com"
-    SHARED_VPC:
-      HOST_PROJECT_ID: "example-prod-networks"
-      SUBNETS:
-      - NAME: example-subnet
+    PROJECT:
+      PROJECT_ID: "example-prod-apps"
+      APIS:
+      - "container.googleapis.com"
+      SHARED_VPC:
+        HOST_PROJECT_ID: "example-prod-networks"
+        SUBNETS:
```

Comment:
could you remind me if this creates the subnet or not?

Reply:
It only adds permissions on the subnet for this project's servic accounts, the subnet itself is in the network project.